File: standard/fizzbuzz.c
#include<stdio.h>

int main(void){
  int num = 1;
  char str[12];
  for(num=1;num<=100;num++){
    sprintf(str,"%d",num);
    printf("%s\n",(num%3==0 && num%7==0) ? "FizzBuzz!" : (num%3==0 ? "Fizz" : (num%7==0 ? "Buzz" : str)));
  }
}

File: standard/fizzbuzz.js
var num = 1;
while(1){
  document.write(num%3==0 ? (num%7==0 ? "FizzBuzz!<br />" : "Fizz<br />") : (num%7==0 ? "Buzz<br />" : num+"<br />") );
  num++;
  if(num==101) break;
}
